packages/competence/design/competency-master-index.md

```
@@ -75,6 +75,14 @@ Each family, in practice, draws its own Expertise + I1 plus whichever shared can
 
 ---
 
+**E2 — SE.ARCHITECTURE specialization**
+| Code | Name |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)